src/middlewared/middlewared/api/v25_04_0/docker.py

```diff
@@ -0,0 +1,59 @@
+import typing
+
+from pydantic import conint, IPvAnyInterface, field_validator
+
+from middlewared.api.base import (
+    BaseModel, Excluded, excluded_field, ForUpdateMetaclass, NonEmptyString, single_argument_args,
+)
+from middlewared.plugins.docker.state_utils import Status
```

Comment:
We can't do this since API must be immutable. If you change `Status` definition, API definition will change.